src/main/java/com/duckblade/osrs/toa/features/scabaras/overlay/MatchingPuzzleSolver.java

```diff
@@ -49,6 +51,19 @@ public class MatchingPuzzleSolver implements PluginLifecycleComponent
 		.put(45373, Color.green) // boot
 		.build();
 
+	private static final Map<Integer, Integer> TILE_NUMBER = ImmutableMap.<Integer, Integer>builder()
+		// these are intentionally out of id order since line, crook, hand, bird are always auto-completed in solos
+		.put(45365, 1) // line
+		.put(45367, 2) // crook
+		.put(45369, 3) // hand
+		.put(45371, 4) // bird
+		.put(45366, 5) // knives
+		.put(45368, 6) // diamond
+		.put(45370, 7) // star
+		.put(45372, 8) // wiggle
+		.put(45373, 9) // boot
+		.build();
```

Comment:
Re-ordered these so that 1, 2, 3, 4 are the ones completed by default in solos, and you do 5, 6, 7, 8, 9, rather than 2, 4, 6, 8, 9.